core/src/main/resources/templates/index.html

```diff
@@ -440,15 +440,6 @@ <h4 class="mb-3">Shortcuts</h4>
             <!-- ============================================================== -->
             <!-- Bread crumb and right sidebar toggle -->
             <!-- ============================================================== -->
-            <div class="row page-titles">
-                <div class="row" ng-if="dashboardDetails.coralAvailableForUser === 'true' && userrole === 'SUPERADMIN'">
-                    <div class="ribbon-wrapper card mt-4">
-                        <div class="ribbon ribbon-success">Explore the new user interface</div>
-                        <p class="ribbon-content">
-                            You're currently logged in as superadmin. To experience the new user interface, switch to your user account.
-                        </p>
-                </div>
-            </div>
```

Comment:
this needs a bit of adjusting, its left the landing page in react "smooshed"

Reply:
❓  what's where smooshed? :D

Reply:
The superadmin old ui dashboard landing page was smooshed.

I think it just needs those css updates we discussed yesterday